sdk/tests/conformance/extensions/webgl-compressed-texture-astc.html

```diff
@@ -1678,9 +1932,104 @@
     // Test that texture upload buffer size is validated correctly.
     ctu.testFormatRestrictionsOnBufferSize(gl, validFormats, expectedByteLength, getBlockDimensions);
 
-    // Tests ASTC texture with every format
-    testLDRTextures();
-    testHDRTextures();
+    // TODO: remove after http://crbug.com/1049806 is fixed
+    ext.getSupportedProfiles = ext.getSupportedProfiles || (() => ["ldr", "hdr"]);
```

Comment:
getSupportedProfiles is added to chrome (simply returns ["ldr"] for now). I think it is safe to remove it.

Reply:
Without HDR profile support, `TEXTURE_3D` target must generate `INVALID_OPERATION`. This is already implemented in ANGLE and covered in WebGL tests by this PR. To pass this test, Chromium has to accurately report supported profiles.